README.md

```diff
@@ -33,7 +33,14 @@ be provided via config/<provider>.json and needs to be updated when that informa
 
 ## Available providers
 
-Currently, a couple of providers are supported: deer, which uses Fleetster as backend provider, Lastenvelo Freiburg, VOI Karlsruhe via backend provider Raumobil, and my-e-car and Stadtmobil Südbaden via backend provider IXSI.
+Currently, a couple of providers are supported:
+
+* deer, which uses Fleetster as backend provider,
+* Lastenvelo Freiburg
+* VOI Karlsruhe via backend provider Raumobil
+* Stadtwerk Tauberfranken via backend provider MOQO
+* Carsharing Südtiro via NOI's OpenDataHub
```

Comment:
```suggestion
* Carsharing Südtirol via NOI's OpenDataHub
```

Reply:
Added in [8bf902c](https://github.com/mobidata-bw/x2gbfs/pull/86/commits/8bf902c3d1724cbc88048818fba3386925fe71d6)